Yoast/wordpress-seo
# Update the new logo's in the plugin

## Summary

## Relevant technical choices:

* Reduced the image sizes to the size they will be actually displayed on, greatly improving the sharpness of the images.

## Test instructions

This PR can be tested by following these steps:

* (See Issue)
* http://local.wordpress.test/wp-admin/admin.php?page=wpseo_licenses
* http://local.wordpress.test/wp-admin/admin.php?page=wpseo_configurator
* Open any Help Center and go to the `Support` tab to see the new logo.

## Quality assurance


Fixes #9047
